app.js

```diff
@@ -6,18 +6,70 @@ var port = process.env.port || 3000;
 app.set("view engine", "ejs");
 app.use(express.static("public"));
 
-app.get("/", function(req, res) {
-    request("https://v2.jokeapi.dev/joke/Any", function(err, response, body) {
-        if(err) {
-            console.log(err);
+app.get("/getJokes/:type", function (req, res) {
+  const jokeType = req.params.type;
+  if (jokeType === "dadJokes") {
+    request(
+      {
+        method: "GET",
+        url: "https://dad-jokes.p.rapidapi.com/random/joke",
+        headers: {
+          "x-rapidapi-host": "dad-jokes.p.rapidapi.com",
+          "x-rapidapi-key":
+            "0fd4d724ffmsh4ea8edb72ad20bdp186caejsn8a504fc2f046",
+          useQueryString: true,
+        },
+      },
+      function (err, response, body) {
+        if (err) {
+          console.log(err);
         } else {
-            data = JSON.parse(body);
-            res.render("getJokes",{data: data});
+          data = {
+            body: JSON.parse(body).body[0],
+            urlName: "dadJokes",
+          };
+          res.render("getJokes", { data: data });
         }
-    })
+      }
+    );
+  } else if (jokeType === "chuck") {
```

Comment:
What if the type in URL is given value other than dadJokes and chuck? Add an else condition to handle this case, where you show message like choose valid values like dadjokes and chuck(show buttons/anchor tags that take the user to that page.